### DISCUSSION TITLE
Import SkySafari observing lists into Stellarium

### DISCUSSION THREAD

**Discussion by markusd112** [Stellarium/stellarium#2308] [Q&A]
Hi,

I have several observing lists in SkySafari 6/7. Is there any simple way to import them into Stellarium?

A SkySafari observing list is simply a text file like this:

SkySafariObservingListVersion=3.0
SortedBy=Default Order
SkyObject=BeginObject
	ObjectID=4,0,89
	CatalogNumber=M 35
	CatalogNumber=NGC 2168
	DefaultIndex=0
EndObject=SkyObject
SkyObject=BeginObject
	ObjectID=4,0,523
	CatalogNumber=NGC 2158
	DefaultIndex=1
EndObject=SkyObject
SkyObject=BeginObject
	ObjectID=4,0,2248
	CatalogNumber=IC 443
	CatalogNumber=LBN 844
	DefaultIndex=2
EndObject=SkyObject
SkyObject=BeginObject
	ObjectID=4,0,22
	CommonName=Beehive Cluster
	CommonName=Praesepe
	CatalogNumber=M 44
	CatalogNumber=NGC 2632
	DefaultIndex=3
EndObject=SkyObject

Thanks,

Markus

**Comment by gzotti** [comment]
There is no direct import. Stellarium's ObservingLists are still undergoing development. You could however analyze the structure of an observing list and write a converter. Currently I won't know what to do with the DefaultIndex or ObjectID entries.

**Reply by markusd112** [reply]
Thanks for your reply. I think some converting script could simply use the "CatalogNumber", but I am not familiar with the json structure of Stellarium and which parameteres are mandatory:

{
    "defaultListUuid": "{f557e147-c0f2-4889-8565-22d3f9140c0b}",
    "observingLists": {
        "{f557e147-c0f2-4889-8565-22d3f9140c0b}": {
            "description": "",
            "jd": "2459639.319037",
            "location": "xxxxxxx, Western Europe",
            "name": "Fotografie",
            "objects": [
                {
                    "designation": "NGC 2359"
                }
            ],
            "sorting": ""
        }
    },
    "shortName": "Observing lists for Stellarium",
    "version": "1.0"
}

Maybe some CSV import functionality would be helpful in Stellarium for such purpose...

**Comment by jwilliams1976** [comment]
Is anyone working on this? It has become a major issue as Sky Safari has seemingly become the de facto standard for end users. As such every astronomy website seems to have Sky Safari observing lists available but nothing for Stellarium. It should be a fairly simple feature as it's just matching the correct fields between the different formats.

**Reply by gzotti** [reply]
No. Anybody in need of this feature and familiar with both programs could provide it.

**Comment by jwilliams1976** [comment]
Thanks. I may give it a stab.
